bgpd/bgp_trace.h

```diff
@@ -43,7 +43,7 @@ TRACEPOINT_EVENT_CLASS(
 	packet_process,
 	TP_ARGS(struct peer *, peer, bgp_size_t, size),
 	TP_FIELDS(
-		ctf_string(peer, peer->host ? peer->host : "(unknown peer)")
+		ctf_string(peer, PEER_HOSTNAME(peer))
```

Comment:
I am a bit confused by the need for this test.  I see about a bajillion places we just have peer->host and use it without checking for NULL.  Why do we have the need here but not in all those other places?

Reply:
It's null sometimes. I don't remember where I saw it be null but it can be null. I'd rather do the nullity check than leave an implicit condition in the code that the tracepoints that don't null check it are only used at points where a certain struct field is null.